scripts/ticketmaster-API.py

```diff
@@ -35,8 +38,10 @@
 date_list = list(map(lambda x: [x.strftime("%Y-%m-%d"+"T00:00:00Z"),x.strftime(
     "%Y-%m-%d"+"T23:59:59Z")],date_range))
 
+entry = 0
 
 for us_state in us_states:
+    print(us_state)
```

Comment:
[RC]
Remember to remove print statements :)

Reply:
Yeah, my bad; removed